components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/OIDProviderConfigResponse.java

```diff
@@ -81,6 +83,10 @@ public class OIDProviderConfigResponse {
     private String deviceAuthorizationEndpoint;
     private String webFingerEndpoint;
     private Boolean tlsClientCertificateBoundAccessTokens;
+    private String mtlsTokenEndpoint;
+    private String mtlsPushedAuthorizationRequestEndpoint;
+
+    private static final String MUTUAL_TLS_ALIASES_ENABLED = "OAuth.MutualTLSAliases.Enabled";
```

Comment:
Do we need a configuration to enable /disable MTLS? For SaaS deployment can't ee depend on the existing config which we use to control other SaaS specific properties?

Reply:
There's no backend config to identify a SaaS deployment. Hence added a this config to enable MTLS aliases in the well known response